test/unit/helper/fieldValue.js

```diff
@@ -0,0 +1,41 @@
+const field = require('../../../helper/fieldValue');
+
+module.exports.tests = {};
+
+module.exports.tests.getStringValue = function(test) {
+  test('getStringValue', function(t) {
+    t.equal(field.getStringValue([]), '', 'empty array');
+    t.equal(field.getStringValue(''), '', 'empty string');
+    t.equal(field.getStringValue('foo'), 'foo', 'string');
+    t.equal(field.getStringValue(['foo','bar']), 'foo', 'array');
+    t.equal(field.getStringValue(['']), '', 'array with empty string');
+    t.equal(field.getStringValue(-0), '-0', 'number');
+    t.equal(field.getStringValue(+0), '0', 'number');
+    t.equal(field.getStringValue({foo: 'bar'}), '[object Object]', '_.toString');
```

Comment:
add a comment that this is not necessarily desired behavior, just how it used to work